Client/Frontend/Browser/BraveWebView.swift

```diff
@@ -46,4 +46,31 @@ class BraveWebView: WKWebView {
         lastHitPoint = point
         return super.hitTest(point, with: event)
     }
+
+    private func generateJavascriptFunctionString(functionName: String, args: [Any]) -> String {
+        let argsJS = args
+          .map { "'\(String(describing: $0).escapeHTML())'" }
+          .joined(separator: ", ")
+        return "\(functionName)(\(argsJS))"
+    }
+
+    func evaluateSafeJavascript(functionName: String, args: [Any], completion: @escaping ((Any?, Error?) -> Void)) {
+        let javascript = generateJavascriptFunctionString(functionName: functionName, args: args)
+        evaluateJavaScript(javascript) { data, error  in
+            completion(data, error)
+        }
+    }
+}
+
+extension String {
+    /// Encode HTMLStrings
+    func escapeHTML() -> String {
+       return self
+        .replacingOccurrences(of: "&", with: "&amp;", options: .literal, range: nil)
+        .replacingOccurrences(of: "\"", with: "&quot;", options: .literal, range: nil)
+        .replacingOccurrences(of: "'", with: "&#39;", options: .literal, range: nil)
+        .replacingOccurrences(of: "<", with: "&lt;", options: .literal, range: nil)
+        .replacingOccurrences(of: ">", with: "&gt;", options: .literal, range: nil)
+        .replacingOccurrences(of: "`", with: "&lsquo;", options: .literal, range: nil)
```

Comment:
`, range: nil` can still be removed here for each line but I'm approving anyways, as its a change that can be bundled with something later

Reply:
fixing, missed this :)